Doc/library/venv.rst

```diff
@@ -168,31 +168,31 @@ creation according to their needs, the :class:`EnvBuilder` class.
     The :class:`EnvBuilder` class accepts the following keyword arguments on
     instantiation:
 
-    * ``system_site_packages`` -- a Boolean value indicating that the system Python
+    * *system_site_packages* -- a boolean value indicating that the system Python
```

Comment:
We could also consider using a parameter list for this (which would render similar to https://docs.python.org/3/library/sqlite3.html#sqlite3.connect). But it's okay if you want to declare that out of scope for this PR :-)